pages/Configuring/Dispatchers.md

```diff
@@ -85,6 +85,14 @@ layout pages (See the sidebar).
 
 {{< callout type=warning >}}
 
+uwsm users should avoid using `exit` dispatcher, as exiting Hyprland this way causes various systemd-related problems after restarting the compositor, such xdg-desktop-portal-hyprland and other services do not restart. Use `loginctl terminate-user ""` instead.
+
+It's also strongly adviced to replace `exit` dispatcher with `loginctl terminate-user ""` inside `hyprland.conf` keybinds section.
```

Comment:
could use a link to https://github.com/Vladimir-csp/uwsm#how-to-stop here

Reply:
I think it's better to just leave the warning, in this